server/src/routes/authenticate.js

```diff
@@ -0,0 +1,69 @@
+import { Router } from 'express';
+const router = Router();
+import isLoggedIn from '../middlewares/isLoggedIn.js';
+
+//
+import session from 'express-session';
+
+router.use(session({
+  resave: false,
+  saveUninitialized: true,
+  secret: 'SECRET'
+}));
+//
+
+//
+import passport from 'passport';
+var userProfile;
+
+router.use(passport.initialize());
+router.use(passport.session());
+
+router.get('/success', isLoggedIn, (req, res) => {
+  // TODO: render success login page
+
+  res.send(userProfile);
+});
+
+router.get('/error', (req, res) => {
+  // TODO: render error login page
+
+  res.send('error logging in');
+});
+
+passport.serializeUser(function(user, cb) {
+  cb(null, user);
+});
+
+passport.deserializeUser(function(obj, cb) {
+  cb(null, obj);
+});
+//
+
+//
+import GoogleStrategy from 'passport-google-oauth';
+const GOOGLE_CLIENT_ID = '384846276379-ih4ucc8boll6b829up2vcrsiaolvirc2.apps.googleusercontent.com';
+const GOOGLE_CLIENT_SECRET = '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';
```

Comment:
Can we get this from the configs and make .env variables for them?

Reply:
Yes, we should. This is for test purposes only. Also we should discuss what google account will be used and with what permissions.